.github/workflows/build-and-push-rag-content.yaml

```diff
@@ -21,6 +21,7 @@ on:
 env:
   IMAGE_NAME: quay.io/openstack-lightspeed/${{ github.event.repository.name }}
   OS_VERSION: 2024.2
+  TAGS: latest
```

Comment:
As discussed, I would set a different default that is not meaningful, since our current builds are explicitly covered later on in the file.

Reply:
Done, thanks!:)